scripts/generate-oas.ts

```diff
@@ -57,6 +57,21 @@ export function getAPIDetailsFromExchange(directory: string): ApiSpecDetail {
     const exchangeConfig = fs.readJSONSync(
       exchangePath
     ) as download.ExchangeConfig;
+
+    // Special handling for shopper-baskets-v2
+    if (
+      exchangeConfig.assetId === 'shopper-baskets-oas' &&
```

Comment:
Can we be a bit generic for the apis ? If other apis happen to have version 2, we have to come back and manually fix this?